### ISSUE TITLE
Import facebook friends from json output

### ISSUE THREAD

**Issue by lucidlogic** [monicahq/monica#2221]
From the facebook settings you can download your friend information as json
1) go here https://www.facebook.com/settings?tab=your_facebook_information
2) view 'Download Your Information' 
![image](https://user-images.githubusercontent.com/932221/50508667-7fa89f80-0a8b-11e9-9319-7342edc5f1c8.png)
3) Deselect All
![image](https://user-images.githubusercontent.com/932221/50508682-8f27e880-0a8b-11e9-9bab-ae4345688584.png)
4) select friends
![image](https://user-images.githubusercontent.com/932221/50508694-9c44d780-0a8b-11e9-9ede-106e5b66f452.png)
5) choose json as the output
![image](https://user-images.githubusercontent.com/932221/50508718-aebf1100-0a8b-11e9-9404-913ed6e1b0e1.png)
6) Create File
![image](https://user-images.githubusercontent.com/932221/50508731-bb436980-0a8b-11e9-80a4-a3af9c015a6e.png)

**Comment by lucidlogic** [comment]
We could do a similar thing with messages

**Comment by marekdlugos** [comment]
This is a deal breaker for me in order to use Monica! Really needed!

**Comment by Alex-Poon** [comment]
Facebook's Graph API no longer provides friends_list data. The unethical route would be to use a facebook scraper. It may be worth exploring if using a chrome extension that captures this data is legal.

**Comment by marekdlugos** [comment]
Right, that's what I did. Installed this [Toolkit for FB Chrome Extension](https://chrome.google.com/webstore/detail/toolkit-for-fb/fcachklhcihfinmagjnlomehfdhndhep) which allowed me to get a very nice piece of information about all my friends in one click:

```
  {
    "uid": number,
    "photo": path_to_profile_pic,
    "path": path_to_profile,
    "fullName": ...,
    "firstname": ...,
    "lastname": ...
  },
```

Seems like this is the way to go.
